contracts/VaultV3.vy

```diff
@@ -106,21 +106,21 @@ API_VERSION: constant(String[28]) = "0.1.0"
 
 # STORAGE #
 strategies: public(HashMap[address, StrategyParams])
-balanceOf: public(HashMap[address, uint256])
+balance_of: public(HashMap[address, uint256])
```

Comment:
every storage variable should be internal and we need a getter for the public ones